packages/nextjs/pages/project.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
```

Comment:
I think several of the remaining files were accidentally included from when you made your last set of changes. They will need to be removed. 

Normally when you start a new task you should get the latest from the `main` branch and it isn't advised to add new commits directly to your `main` branch. Instead create a new branch and make changes there. That way your local `main` branch doesn't get cluttered with changes that may never be merged in. Here is the best way to get a fresh reset of your main branch 
```
git pull upstream main
git reset --hard upstream/main
```

Now your local main branch should be fresh. What about all of your changes? They still exist in your remote main branch (your branch, not upstream). All you need to do is pull in each commit you want (avoiding any you don't want).
```
git cherry-pick <commit-hash>
```
I think the only commit you will need is 8aa736a5d7264f40ebb2d93e42d3fae05a247a7c and then you will need to remove the deployedContracts.ts changes.

Now, before you commit these changes you should create a new branch `git checkout -b my-new-branch` and then you can stage and commit your changes then push your new branch to your remote repo. Once you have done that, create a new PR and remove this one.

Let me know if you need help with any of those steps.